### ISSUE TITLE
Can you add the weight information into link.dat.test?

### ISSUE THREAD

**Issue by justicevita** [yangji9181/HNE#11]
In some case, We want to split a smaller dataset from your preprocess dataset, but I find that the weight information is missing.So, I think maybe you can add the weight information into link.dat.test?

**Comment by xiaoyuxin1002** [comment]
It is very unfortunate that the link weight information of links in link.dat.test was lost during the data preprocessing step. 

So maybe one alternative way to split a smaller dataset from the preprocessed dataset is to split the given training set link.dat into a smaller training set and a test set with link weight information.
